app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1273,18 +1275,18 @@ $include-html-global-classes: $include-html-classes;
 // $include-html-table-classes: $include-html-classes;
 
 // These control the background color for the table and even rows
-// $table-bg: $white;
-// $table-even-row-bg: $snow;
+$table-bg: rgba( $black, .6 );
```

Comment:
`.6` should be written with a leading zero as `0.6`<br>Expected 0 spaces between parentheses instead of 1